saleor/graphql/webhook/subscription_types.py

```diff
@@ -1599,6 +1599,7 @@ class TransactionAction(SubscriptionObjectType, AbstractType):
     amount = PositiveDecimal(
         description="Transaction request amount. Null when action type is VOID.",
     )
+    currency = graphene.String(description="Currency code.")
```

Comment:
Can you add `required=True` and label `ADDED_IN_316` to the description?
Something like:
`currency = graphene.String(description="Currency code." + ADDED_IN_316, required=True)`

Reply:
Ok. I just forgot about that. Will add and update the PR ASAP.